src/archivematicaCommon/lib/archivematicaFunctions.py

```diff
@@ -200,3 +200,21 @@ def create_structured_directory(basepath, manual_normalization=False, printing=F
     create_directories(REQUIRED_DIRECTORIES, basepath=basepath, printing=printing)
     if manual_normalization:
         create_directories(MANUAL_NORMALIZATION_DIRECTORIES, basepath=basepath, printing=printing)
+
+
+def close_db_connections(fn):
+    """
+    Decorator to wrap a threaded function in a try-catch ensuring that the
+    database connections are closed if unrecoverable errors have occurred or if
+    it outlived its maximum age.
+
+    Similar to Django's behavior within HTTP request cycles.
+    """
+    def wrapped(*args,  **kwargs):
+        from django.db import close_old_connections
```

Comment:
That was intentional because I was trying to make sure that the module is not being cached by Python across multiple threads... but maybe that's going to happen regardless where the import statement is? I'll dig into it and see if I can put the import in the expected place.